### PR TITLE
changed sdk download link and added a canary update

### PR DESCRIPTION
**Pull request by fehersanyi-bitrise**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -48,57 +50,59 @@ RUN sdkmanager "emulator" "tools" "platform-tools"
 # The `yes` is for accepting all non-standard tool licenses.
 
 # Please keep all sections in descending order!
+
+RUN sdkmanager --update --channel=3
```
Comment (koral--) [inline Dockerfile]: `yes` is probably missing here

File: Dockerfile [hunk 2]
```diff
@@ -38,7 +38,9 @@ ENV PATH ${PATH}:${ANDROID_HOME}/tools:${ANDROID_HOME}/tools/bin:${ANDROID_HOME}
 # Accept licenses before installing components, no need to echo y for each component
 # License is valid for all the standard components in versions installed from this file
 # Non-standard components: MIPS system images, preview versions, GDK (Google Glass) and Android Google TV require separate licenses, not accepted there
-RUN yes | sdkmanager --licenses
+RUN yes | sdkmanager  --licenses
+
+RUN touch /root/.android/repositories.cfg
 
 # Platform tools
 RUN sdkmanager "emulator" "tools" "platform-tools"
```
Comment (koral--) [inline Dockerfile]: This should be probably below line with `--update`

File: Dockerfile [hunk 3]
```diff
@@ -21,7 +21,7 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get install -y openjdk-8-jdk libc6:i386 l
 # --- Download Android SDK tools into $ANDROID_HOME
 
 RUN cd /opt \
-    && wget -q https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip -O android-sdk-tools.zip \
+    && wget -q https://dl.google.com/android/repository/sdk-tools-linux-3859397.zip -O android-sdk-tools.zip \
```
Comment (koral--) [inline Dockerfile]: Why initial sdk tools version is downgraded 🤔

Comment (fehersanyi-bitrise) [reply Dockerfile]: valid question, I was blindly trying things out 🤔 changed back and rebuilding to see if it was totally useless

File: Dockerfile (hunk 1 above)
Comment (fehersanyi-bitrise) [reply Dockerfile]: trying with the next commit

File: Dockerfile (hunk 2 above)
Comment (fehersanyi-bitrise) [reply Dockerfile]: trying with the next commit

**Pull request merged**